Code/Tools/SceneAPI/SceneData/Tests/SceneManifest/SceneScriptRuleTests.cpp

```diff
@@ -16,12 +16,15 @@
 #include <SceneAPI/SceneData/Rules/CoordinateSystemRule.h>
 #include <SceneAPI/SceneData/Behaviors/ScriptProcessorRuleBehavior.h>
 #include <SceneAPI/SceneCore/Events/ExportProductList.h>
+#include <SceneAPI/SceneCore/Mocks/MockBehaviorUtils.h>
 
 #include <AzCore/Math/MathReflection.h>
 #include <AzCore/Math/Quaternion.h>
 #include <AzCore/Name/NameDictionary.h>
 #include <AzCore/RTTI/BehaviorContext.h>
+#include <AzCore/RTTI/ReflectContext.h>
```

Comment:
Nit: It seems that AzCore/RTTI/ReflectContext.h and AzCore/Script/ScriptContextAttributes.h has already been included in SceneAPI/SceneCore/Mocks/MockBehaviorUtils.h and you may not need to include them again here.